### PR TITLE
add dbotscore outputs path

### PR DESCRIPTION
**Pull request by jochman**
added outputPaths.dbotscore to commonserver

### CODE REVIEW

File: Scripts/script-CommonServer.yml [hunk 1]
```diff
@@ -1410,7 +1410,8 @@ script: |-
       url: 'URL(val.Data && val.Data === obj.Data)',
       domain: 'Domain(val.Name && val.Name === obj.Name)',
       cve: 'CVE(val.ID && val.ID === obj.ID)',
-      email: 'Account.Email(val.Address && val.Address === obj.Address)'
+      email: 'Account.Email(val.Address && val.Address === obj.Address)',
+      dbotscore: 'DBotScore(val.Indicator === obj.Indicator && val.Vendor === obj.Vendor)',
```
Comment (Itay4) [inline Scripts/script-CommonServer.yml]: Please verify it exists first by `val.Indicator && val.Indicator ===...`

Comment (aviadl) [reply Scripts/script-CommonServer.yml]: DBotScore path should be:
`DBotScore` and that is it

**Comment by ronykoz** [comment]
The id_set.json change is because of a problematic way i write the id_set, should be ok once Anar will merge my PR

File: Scripts/script-CommonServer.yml (hunk 1 above)
Comment (jochman) [reply Scripts/script-CommonServer.yml]: It's already exists in CommonServerPython, I just added it to CommonServer. shouldn't we use it?

**Pull request closed without merging**